ap/plotting/plotter.py

```diff
@@ -14,45 +14,46 @@
 mplhep = maybe_import("mplhep")
 
 
-def draw_error_stairs(ax: plt.Axes, h: hist.Hist, kwargs: Optional[dict] = None) -> None:
-    if kwargs is None:
-        kwargs = {}
-    norm = kwargs.pop("norm", 1)
-    values = h.values() / norm
-    error = np.sqrt(h.variances()) / norm
+def draw_error_stairs(ax: plt.Axes, h: hist.Hist, norm: float = 1.0, **kwargs) -> None:
+    # compute relative errors
+    rel_error = h.variances()**0.5 / h.values()
+    rel_error[np.isnan(rel_error)] = 0.0
+
+    # compute the baseline
+    # fill 1 in places where both numerator and denominator are 0, and 0 for remaining nan's
+    baseline = h.values() / norm
+    baseline[(h.values() == 0) & (norm == 0)] = 1.0
+    baseline[np.isnan(baseline)] = 0.0
+
     defaults = {
-        "edges": h.axes[0].edges,
-        "baseline": values - error,
-        "values": values + error,
+        "x": h.axes[0].centers,
+        "width": h.axes[0].edges[1:] - h.axes[0].edges[:-1],
+        "height": baseline * 2 * rel_error,
+        "bottom": baseline * (1 - rel_error),
         "hatch": "///",
         "facecolor": "none",
         "linewidth": 0,
         "color": "black",
```

Comment:
The hatch is not drawn for some reason. Changing `"facecolor: "black"` would at least make the error bands visible, but that's not really the style we are looking for.

Reply:
Yup, see above :)